src/assets/translations/knb.json

```diff
@@ -24850,7 +24850,7 @@
                   "verse": "4"
                 },
                 {
-                  "text": "Mondjátok Sion leányának: Íme, királyod jön hozzád; szelíd ő, s szamárháton ül, szamárcsikón, teherhordó állat fián«.",
+                  "text": "»Mondjátok Sion leányának: ‘Íme, királyod jön hozzád; szelíd ő, s szamárháton ül, szamárcsikón, teherhordó állat fián’«.",
```

Comment:
Taking to the correct form which seen several places in the KNB Bible e.g: KNB [lk19,30 - lk19,31].
The only difference to this part is that we do not fix, but keep the ordering of the ending characters: 
’«.
(see the general comment on these endings in knb)

Reply:
Documented